src/mocks/drag-event-creator.spec.ts

```diff
@@ -0,0 +1,13 @@
+export class DragEventCreator {

```

Comment:
Rework this helperclass (rename, replace, combine data transfer property, try to remove spy)